src/Microsoft.OpenApi.OData.Reader/Generator/OpenApiErrorSchemaGenerator.cs

```diff
@@ -27,28 +27,40 @@ internal static class OpenApiErrorSchemaGenerator
         public static IDictionary<string, OpenApiSchema> CreateODataErrorSchemas(this ODataContext context)
         {
             Utils.CheckArgumentNull(context, nameof(context));
+            var rootNamespaceName = context.GetErrorNamespaceName();
 
             return new Dictionary<string, OpenApiSchema>()
             {
-                // odata.error
-                { "odata.error", CreateErrorSchema() },
-
-                // odata.error.main
-                { "odata.error.main", CreateErrorMainSchema() },
-
-                // odata.error.detail
-                { "odata.error.detail", CreateErrorDetailSchema() },
-
-                // odata.error.innererror
-                { "odata.error.innererror", CreateInnerErrorSchema(context) }
+                { $"{rootNamespaceName}{OdataErrorClassName}", CreateErrorSchema(rootNamespaceName) },
+                { $"{rootNamespaceName}{MainErrorClassName}", CreateErrorMainSchema(rootNamespaceName) },
+                { $"{rootNamespaceName}{ErrorDetailsClassName}", CreateErrorDetailSchema() },
+                { $"{rootNamespaceName}{InnerErrorClassName}", CreateInnerErrorSchema(context) }
             };
         }
+        internal const string OdataErrorClassName = "ODataError";
```

Comment:
Out of curiosity: Why did you choose to define the `const` values below where they are first used?

Reply:
because it doesn't matter to the compiler, or to me. I use symbol navigation rather than scrolling when I browse code (either f12 or ctrl + P and then type @ in vs code)

Reply:
I think for uniformity purposes (see [example](https://github.com/microsoft/OpenAPI.NET.OData/blob/f72545004b24a650dfc1d885d0e85bd4656d22ff/src/Microsoft.OpenApi.OData.Reader/Operation/ODataTypeCastGetOperationHandler.cs?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L43-L51)) with other sections of the code base and for easier reading (by other maintainers) you could stick to defining these before they are first used?

Reply:
changed. Give a try to symbol navigation instead of scrolling, you'll see it's much faster!

Reply:
Thank you for taking in this suggestion.